### ISSUE TITLE
Links in info admonitions are indistinguishable

### ISSUE THREAD

**Issue by phoqe** [elviswolcott/remark-admonitions#19]
Hello, I’ve found that `info` admonitions that have links are indistinguishable. Here’s an example:
![image](https://user-images.githubusercontent.com/7033377/75537356-25042e00-5a17-11ea-84fc-d990523480e4.png)

**Comment by elviswolcott** [comment]
Good catch! I’ll look into the best way to resolve this while maintaining
infima styling and get a new version out. I’m traveling right now so it
might take a day.

**Comment by phoqe** [comment]
Alright, cool. Safe travels!

**Comment by yangshun** [comment]
@elviswolcott I could perhaps make a PR to address this. I've addressed this issue in Infima but it seems like this plugin forked the styles and used different classes naming, so the improvements in Infima didn't get picked up here.

**Comment by elviswolcott** [comment]
That would be great! Due to a suggestion (forget who from) in the PR on
docusuaurs it’s setup so it should be easy to change which infima variables
the plugin variables use.

**Comment by elviswolcott** [comment]
@yangshun I'm not seeing anything related to this on the latest version of `infima.min.css`. Am I missing something? Do you still want to take this?

**Comment by yangshun** [comment]
Sorry I don't really have time to work on it yet. If you have the time please go ahead.

I've added special handling for links but that's under the `.alert` class name, not `.admonition`. I hope that your library can rename the classes and then it would work automatically. Ideally this library doesn't provide the classes because that would be duplicating work with Infima.

**Comment by elviswolcott** [comment]
#20 should fix this, if it looks good (visually) I'll merge and release a new version
